mlflow/models/model.py

```diff
@@ -109,6 +127,21 @@ def saved_input_example_info(self, value: Dict[str, Any]):
         # pylint: disable=attribute-defined-outside-init
         self._saved_input_example_info = value
 
+    def get_model_info(self) -> ModelInfo:
+        """Create a ModelInfo instance that contains the model metadata."""
+        return ModelInfo(
+            run_id=self.run_id,
+            artifact_path=self.artifact_path,
+            model_uri="runs:/{}/{}".format(self.run_id, self.artifact_path),
```

Comment:
Let's use `artifact_utils.get_artifact_uri` to generate uri.

Reply:
I think the `runs:/` scheme is easy to read (shorter) and also consistent with our examples in the documentation ([example](https://mlflow.org/docs/latest/python_api/mlflow.pytorch.html#mlflow.pytorch.load_model)), while the absolute artifact uri returned by `get_artifact_uri` is much longer.
Would you mind elaborating on the reasons for using `get_artifact_uri`?

Reply:
why not add a helper method in `artifact_utils` modules for this ?

Reply:
I think the new helper method will not add much value because 1) it's a pretty simple format to manually type it. 2) the string literal is mentioned in many places in our document, so it's easy for the user to discover and understand, while the proposed helper function will need an extra item in our documentation, which seems to be less discoverable.